frontend/pages/[lng]/auth/clients.tsx

```diff
@@ -260,7 +261,9 @@ const Clients = () => {
               <td>{client.client_id}</td>
               <td>{client.redirect_uri}</td>
               <td>
-                <a href={`client-details?id=${client.client_id}`}>View</a>
+                <Link href={`client-details?id=${client.client_id}`}>
```

Comment:
Use our own `LangLink` that wraps `Link` and adds the correct language path if necessary; also you don't need to pass anything for the `a` below as it's passed on